spec/javascript-spec.coffee

```diff
@@ -1138,6 +1220,12 @@ describe "Javascript grammar", ->
       {tokens} = grammar.tokenizeLine('this.obj.prototype = new El()')
       expect(tokens[0]).toEqual value: 'this', scopes: ['source.js', 'variable.language.js']
 
+      {tokens} = grammar.tokenizeLine('$this')
+      expect(tokens[0].value).toEqual '$this'
```

Comment:
@50Wliu Can you explain to me *exactly* what it is you want me to do? I'm really not following here.